components/Testimonies/Testimonies.js

```diff
@@ -8,7 +9,52 @@ const Testimonies = () => {
   const testimonies =
     testimoniesResponse.status == "success" ? testimoniesResponse.result : []
 
-  return <TestimoniesTable testimonies={testimonies} />
+  const [filterBy, setFilterBy] = useState(null)
+
+  return (
+    <>
+      <div className="row mt-2">
```

Comment:
We should try to consolidate the top part of this table with the bills table since much of it is repeated. But we can do that when we implement filtering and searching so this is fine for now